lib/version_utils.pm

```diff
@@ -860,7 +863,13 @@ sub get_version_id {
     $args{verid_file} //= '/etc/os-release';
 
     my $cmd = "cat $args{verid_file} | grep VERSION_ID | grep -Eo \"[[:digit:]]{1,}\\.[[:digit:]]{1,}\"";
-    $cmd = "ssh root\@$args{dst_machine} " . "$cmd" if ($args{dst_machine} ne 'localhost');
+    if ($args{dst_machine} ne 'localhost') {
```

Comment:
I would introduce dedicated function instead

Reply:
Why do you think that it would need a dedicated function? It would duplicate the code mostly

Reply:
ok , I thought this function is used in a lot of places but grepping the code proves me wrong . so let's leave it as is